docs/overview/getting_started.rst

```diff
@@ -44,6 +44,25 @@ API throttle limit: We reserve the right to tune the limitations, but they are a
 When the rate limit is exceeded Checkfront will send an HTTP 503 status code.  The number of seconds until the throttle is lifted is sent via the "Retry-After" HTTP header, as specified in RFC 2616.
 
 
+The Public API
+--------------
+Checkfront provides a public API feature that allows un-authenticated users to pull information about items and availability. This data is a reduced version of what you can obtain using the authenticated API, and only provides already public information that could otherwise be accessed by going to the `Customer Booking Page <https://support.checkfront.com/hc/en-us/articles/115004917593-Hosted-Booking-Page>`_.
```

Comment:
Minor, but maybe re-word `otherwise be accessed by going to` to `otherwise be accessed from`

Reply:
Resolved in latest commit